### PR TITLE
:robot: Use builjet arm workers for arm jobs

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:
Runs arm workers on buildjet with native arm64 workers.
Instead of compressing img files, we transform them into qcow2 which are compressed by default, speeding up the build up to 30 minutes.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -614,8 +614,8 @@ arm-image:
     RUN /build-arm-image.sh --use-lvm --model $MODEL --directory "/build/image" /build/$IMAGE_NAME
   END
   IF [ "$COMPRESS_IMG" = "true" ]
-      RUN xz -v /build/$IMAGE_NAME
-      SAVE ARTIFACT /build/$IMAGE_NAME.xz img AS LOCAL build/$IMAGE_NAME.xz
```
Comment (mauromorales) [inline Earthfile:618]: is this change intended? If yes, maybe we can have a different flag to decide whether we want a qcow2 or img

Comment (Itxaka) [reply Earthfile:618]: done!

**Review by mauromorales — DISMISSED** [review]
noiiiice

**Review by mauromorales — APPROVED** [review]

**Comment by jimmykarily** [comment]
This PR only, cost 5 dollars! This doesn't scale :D Let's try to optimize on our workers instead.

**Comment by Itxaka** [comment]
Will resend the patch for not compressing arm images on PRs as separated.

Just 1 PR with 4 or 5 commits already set the usage of buildjet to an estimated 5$ according to the buildjet dashboard. I dont think this scales properly LMAO.
